.env.example

```diff
@@ -0,0 +1,9 @@
+# Interface config
+HOST_IPV4=0.0.0.0:9091 // Default IPv4
```

Comment:
Do we have a default in the code? If this variable is omitted what do we use?

Reply:
Sadly no, we should tho for sure.